docs/platforms/javascript/common/tracing/distributed-tracing/dealing-with-cors-issues/index.mdx

```diff
@@ -16,8 +16,94 @@ notSupported:
   - javascript.nestjs
 ---
 
-If your frontend and backend are hosted on different domains (for example, your frontend is on `https://example.com` and your backend is on `https://api.example.com`), and the frontend does XHR/fetch requests to your backend, you'll need to configure your backend [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Headers) headers to ensure requests aren't blocked.
+If your frontend and backend are hosted on different domains (for example, your frontend is on `https://example.com` and your backend is on `https://api.example.com`), you may need to configure your backend [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Headers) headers to ensure distributed tracing works properly.
```

Comment:
May is incorrect here, you def. Have to do that I think.

Reply:
I think this whole change is incorrect actually, this is not needed to ensure distributed tracing works but to ensure the requests are not blockedm